File: python/EstruturaSequencial/exercicio02.py
base = float (input('Base do retangulo: '))
altura = float (input('Altura do retangulo: '))

area = base * altura
perimetro = base * 2 + altura * 2
diagonal = (base ** 2 + altura ** 2) ** 0.5

print(f'AREA = {area}')
print(f'PERIMETRO = {perimetro}')
print(f'DIAGONAL = {diagonal:.4f}')

File: C++/EstruturaSequencial/Exercicio02.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;

int main() {
    double base, altura;

    cout << "Base do retangulo: ";
    cin >> base;
    cout << "Altura do retangulo: ";
    cin >> altura;

    double area = base * altura;
    double perimetro = 2 * (base + altura);
    double diagonal = sqrt(pow(base, 2) + pow(altura, 2));

    cout << fixed << setprecision(4);
    cout << "AREA = " << area << endl;
    cout << "PERIMETRO = " << perimetro << endl;
    cout << "DIAGONAL = " << diagonal << endl;

    return 0;
}
